src/impi/core.cljs

```diff
@@ -126,6 +126,33 @@
         [w h] (:pixi.render-texture/size texture)]
     (.create js/PIXI.RenderTexture w h mode)))
 
+(defmulti draw-shape! (fn [_ v] (v :pixi.shape/type)))
+
+(defmethod draw-shape! :pixi.shape/circle 
+  [graphics {[x y] :pixi.shape/position
+             radius     :pixi.circle/radius}]
+  (.drawCircle graphics x y radius))
+
+(defmethod draw-shape! :pixi.shape/ellipse 
+  [graphics {[x y]          :pixi.shape/position
+             [width height] :pixi.shape/size}] 
+  (.drawEllipse graphics x y width height))
+
+(defmethod draw-shape! :pixi.shape/polygon 
+  [graphics {path :pixi.polygon/path}] 
```

Comment:
Path is consistent with PIXI naming. More importantly, it is consistent with the pixi implementation details. e.g. If you have a lineStyle it will not cover the border of the polygon only the path drawn by the points.